src/api/cow/index.ts

```diff
@@ -352,10 +357,6 @@ export class CowApi extends BaseApi {
     return finalParams
   }
 
-  protected getApiBaseUrl(): Promise<string> {
-    return super.getApiBaseUrl(this.context.env)
-  }
-
```

Comment:
Most of changes to this file is the linter/prettier/ide sorting imports and methods